setup.py

```diff
@@ -64,7 +64,7 @@
     'Topic :: Security',
     'Topic :: Software Development'
   ],
-  packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
+  packages=["in_toto", "in_toto.models", "in_toto.gpg"],
```

Comment:
One potential downside to this approach is that if you later add a subdirectory or rename a subdirectory, you will have to remember to update `setup.py`.  You might only notice the mistake when users install from release and complain about import errors.

Assuming that Travis doesn't build from a release, continuous integration might not catch the mistake.

Reply:
You have a point there. Maybe the best thing would be to leave the line as it was (recommended by [`setuptools` docs](http://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages)) and instead rename the the `test` directory to `tests`? What do you think?

Reply:
... or revert the commit and remove the trailing `s` from `tests` in the excludes.